RecoTracker/IterativeTracking/python/JetCoreRegionalStep_cff.py

```diff
@@ -254,7 +335,65 @@
+                                         jetCoreRegionalStepSeedsBarrel,
+                                         jetCoreRegionalStepBarrelTrackCandidates,
+                                         jetCoreRegionalStepBarrelTracks,
+                                         #                                   jetCoreRegionalStepClassifier1,jetCoreRegionalStepClassifier2,
+                                         jetCoreRegionalStepBarrel)
+
+JetCoreRegionalStepEndcapTask = cms.Task(jetsForCoreTrackingEndcap,
+                                         firstStepGoodPrimaryVertices,
+                                         #jetCoreRegionalStepClusters,
+                                         jetCoreRegionalStepSeedLayers,
+                                         jetCoreRegionalStepEndcapTrackingRegions,
+                                         jetCoreRegionalStepEndcapHitDoublets,
+                                         jetCoreRegionalStepSeedsEndcap,
+                                         jetCoreRegionalStepEndcapTrackCandidates,
+                                         jetCoreRegionalStepEndcapTracks,
+                                         #                                   jetCoreRegionalStepClassifier1,jetCoreRegionalStepClassifier2,
+                                         jetCoreRegionalStepEndcap)
+
+
+from Configuration.ProcessModifiers.seedingDeepCore_cff import seedingDeepCore
+
+from RecoTracker.FinalTrackSelectors.TrackCollectionMerger_cfi import *
+seedingDeepCore.toReplaceWith(jetCoreRegionalStepTracks, TrackCollectionMerger.clone(
+    trackProducers   = ["jetCoreRegionalStepBarrelTracks",
+                        "jetCoreRegionalStepEndcapTracks",],
+    inputClassifiers = ["jetCoreRegionalStepBarrel",
+                        "jetCoreRegionalStepEndcap",],
+    foundHitBonus    = 100.0,
+    lostHitPenalty   = 1.0
+))
+
+seedingDeepCore.toReplaceWith(jetCoreRegionalStep, jetCoreRegionalStepTracks.clone()) #(*)
+
+## rename jetCoreRegionalStep to e.g. jetCoreRegionalStepImpl
+## alias the MVAValues and QualityMasks products from jetCoreRegionalStepImpl to jetCoreRegionalStep
+#jetCoreRegionalStep = cms.EDAlias(
+#    jetCoreRegionalStepImpl = cms.VPSet(
+#        cms.PSet(type = cms.string("floats")),
+#        cms.PSet(type = cms.string("uchars")),
+#    )
+#)
+## change the alias-from to point to jetCoreRegionalStepTracks
+#seedingDeepCore.toModify(jetCoreRegionalStep,
+#    jetCoreRegionalStepImpl = None,
+#    jetCoreRegionalStepTracks = jetCoreRegionalStep.jetCoreRegionalStepImpl.copy()
+#)
+#
+
+seedingDeepCore.toReplaceWith(JetCoreRegionalStepTask, cms.Task(
+    JetCoreRegionalStepBarrelTask,
+    JetCoreRegionalStepEndcapTask,
+    cms.Task(jetCoreRegionalStepTracks,jetCoreRegionalStep)
+))
+
 fastSim.toReplaceWith(JetCoreRegionalStepTask, 
                       cms.Task(jetCoreRegionalStepTracks,
                                    jetCoreRegionalStep))
```

Comment:
(new comment) 
shouldn't fastsim be supported as well for the new deepcore tracking?
If so, we need fastSim modifiers to handle the barrel-endcap split appropriately.

Reply:
not sure